### ISSUE TITLE
Feature request - modify plan before pushback

### ISSUE THREAD

**Issue by deltazulu44** [skiselkov/BetterPushbackC#319]
It should be possible to change pushback plan as long as puchback process is not commenced / parking break is not released even when tug is already connected.
It's unrealistic to reset the whole tug arriving process to modify the plan.

**Comment by Meyssam120** [comment]
+1

**Comment by flyingtost** [comment]
Yup, i absolutely would love that feature. This would reflect real world ops way better. As IRL the truck is connected and the ac lifted before you usual even ask for a pushback clearance and only then you(the groundcrew) get(s) the instructions on where to push. That what happens in Sim when flying online all the time. Either you wait „for ever“ before commencing push after you did receive your clearance or you risk waiting even longer, as you have preplanned the wrong procedure! :)
 Ideally you would still be able the change the plan, during pushback, as is IRL, but for simplicity i‘d be more than happy to be able to change it up until parking brake release! :)

many thanks for considering it!
Best regards,
Tom

**Comment by zalloon** [comment]
Hi guys, there is a way to accomplish this in its current state... you can assign a key to get the truck connected (independently of having or not a trajectory planned). Then you assign a second key to actually start the pushback, which will trigger the planning screen if you don't have any input yet (after the ATC clearance and instructions). Doing this way, the pushback starts immediately when you command it, as you will have already the truck connected to the ac. Hope it helps you!

**Comment by flyingtost** [comment]
Hi @zalloon,

Wow, thanks for the tipp, i will tried that out later today! :) that would actually help a lot! :)
But fingers crossed, we may still see that feature implemented into the „EFB process“ at some point! :) 

thanks again,
cheers,
Tom
